profiles/base.yaml

```diff
@@ -119,7 +119,8 @@ empty_hunting_room_messages:
 hunting_buddies:
 stop_hunting_if_bleeding: true
 prehunt_buffs:
-
+hunting_room_min_mana: false
```

Comment:
i was trying to simplify by letting me use the hunting_room_min_mana setting as a boolean flag to enter the logic at all. However, I can chagne this to 0 and have the boolean logic be > 0.